scripts/release-notes/gen_ec_release_notes.py

```diff
@@ -317,7 +352,7 @@ def find_new_releases(versions_to_scan, url_base, release_type):
             if nr:
                 new_releases.append(nr)
         except Exception as err:  # pylint: disable=broad-except
-            print(f"WARNING: could not process {release_type} {version}: {err}")
+            logging.error(f"WARNING: could not process {release_type} {version}: {err}")
```

Comment:
Error or warning?

Reply:
Changed to warning and removed `WARNING: ` from msg